src/tzstats/tzstats_reward_provider_helper.py

```diff
@@ -70,9 +84,93 @@ def get_rewards_for_cycle(self, cycle, expected_reward = False, verbose=False):
         resp = resp.json()
 
         for delegator in resp:
+
             if delegator[idx_delegator_address] == self.baking_address:
-                root["delegate_staking_balance"] = int(1e6 * (float(delegator[idx_baker_balance]) + float(delegator[idx_baker_delegated])))
+                root["delegate_staking_balance"] = int(1e6 * (float(delegator[idx_balance]) + float(delegator[idx_baker_delegated])))
             else:
-                root["delegators_balance"][delegator[idx_delegator_address]] = int(1e6 * float(delegator[idx_delegator_balance]))
+                delegator_info = {"staking_balance": 0, "current_balance": 0}
+                delegator_info["staking_balance"] = int(1e6 * float(delegator[idx_balance]))
+                root["delegators_balances"][delegator[idx_delegator_address]] = delegator_info
+
+        #
+        # Get current balance of delegates
+        #
+        # This is done in 2 phases. 1) make a single API call to tzscan, retrieving an array
```

Comment:
This is really a tiny detail: but we should change the comment from tzscan to tzstats

Reply:
ha! fatfinger mistake. still not used to typing tzstats. will fix.